Codes/edit.php

```diff
@@ -99,7 +99,7 @@
                 <?php $cn=$row['course_name']; ?>
 
                 <div class="d-grid gap-2 d-md-block" role="group" aria-label="First group">
-                  <a type="button" class="btn btn-outline-secondary d-print-none" href="coursetable.php">Save</a>
+                  <a type="button" class="btn btn-outline-secondary d-print-none" href="coursetable.php?course=<?php echo $cd; ?>">Save</a>
```

Comment:
No handling mechanism in coursetable.php

Reply:
This will be handled by PurviGoyal7.
First for saving data in database redirection to the saving file will be made then to the coursetable.